src/integration_tests/trafficcontroller/dropsonde_end_to_end_test.go

```diff
@@ -76,10 +77,15 @@ var _ = Describe("TrafficController for dropsonde messages", func() {
 	})
 
 	Context("Firehose", func() {
+
+		var (
+			messages <-chan *events.Envelope
+			errors   <-chan error
```

Comment:
Both `messages` and `errors` are used in the `Streaming` context, as well - maybe they should be defined at a higher level rather than re-defining them?